aggregate/clean_aggregated.py

```diff
@@ -47,4 +57,31 @@ def order_affordable(measures, income) -> List:
         for m in measures:
             rv.append(f"units_affordable_{i}{m}")
             
-    return rv
\ No newline at end of file
+    return rv
+
+
+def rename_col(df: pd.DataFrame, name_mapper: dict, race_mapper: dict, year_mapper: dict, suffix_mapper: dict):
```

Comment:
Similar comment here, could change function name to `rename_col_dcp_pop` or something. Really glad this process is being abstracted to a helper function!